src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +18,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<Machine> getAll(Class<? extends Machine> type);
```

Comment:
Do not use raw types. Always parametrize interfaces and classes if it is possible. The return type of 'getAll' should be parameterized with the generic type 'T' instead of 'Machine'.